scripts/mc_wx.py

```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""This is meant to hold utility scripts for weather
+
+"""
+from __future__ import absolute_import, division, print_function
+
+from hera_mc import cm_utils, mc
+
+wx_options = ['wind_speed', 'wind_gust', 'wind_direction', 'humidity', 'pressure', 'temperature', 'rain']
```

Comment:
I don't think these should be hardcoded here because it duplicates information that's already stored in `hera_mc.weather`. Passing `None` for the `variables` arguments to the weather functions means "use the default list of sensors" so we should use that.

`wx_options` is also used for the `choices` keyword to limit the available value choices for the `--variables` argument but I think it might actually be better not to apply that constraint. You could imagine extending this script to be able to show values from any of the (many thousands) of sensor points that we can fetch from KATCP.